src/Models/Package.php

```
@@ -539,6 +577,28 @@ private static function normalisePackageData(array $data): array
+            'ai_default_provider' => self::enumValue(
+                $data['ai_default_provider'] ?? 'google',
+                self::AI_PROVIDERS,
+                'google'
+            ),
+            'ai_default_model' => self::normaliseDefaultAiModel(
+                $data['ai_default_model'] ?? '',
+                self::enumValue($data['ai_default_provider'] ?? 'google', self::AI_PROVIDERS, 'google')
+            ),
+            'ai_fallback_model' => self::normaliseOpenAiModel(
+                $data['ai_fallback_model'] ?? '',
+                SystemSettings::defaultOpenAiFallbackModel()
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'StratFlow\Models\SystemSettings' in method 'normalisePackageData'. (undefined)

(StaticAccess)